src/proper_types.erl

```diff
@@ -1096,6 +1096,15 @@ function(Arity, RawRetType) when is_integer(Arity), Arity >= 0, Arity =< 255 ->
 function(RawArgTypes, RawRetType) ->
     function(length(RawArgTypes), RawRetType).
 
+%% @doc A map whose keys are defined by the generator `K' and values
+%% by the generator `V'.
+-spec map(K, V) -> proper_types:type() when
```

Comment:
The style of this spec unnecessarily differs from those used in the file.  Please change it to a one-liner for uniformity.

Reply:
Fixed it.